### ISSUE TITLE
can't run grunt from child theme

### ISSUE THREAD

**Issue by titaniumbones** [gpspake/grunterie-child#3]
I'm just  giving the child theme a quick run, and am unalbe to get grunt to run; it gives this error when compiling scss:

```
Warning: ../grunterie/bower_components/foundation/scss/foundation/functions:13: error: error reading values after $modules
```
I'm using an uptodate grunterie and grunterie-child -- have you seen the same issue?

**Comment by gpspake** [comment]
To be honest, I haven't made any changes to this child theme in a while. I'll take a look at it and see if I can get it updated. I already see some things I want to change here.

**Comment by titaniumbones** [comment]
thanks @gspake!

(also awesome to have grunterie updated to F 5.5, thanks for that too).

**Comment by titaniumbones** [comment]
updating versions of node packages in package.json seems to have fixed
the initial compilation errors.  I don't really understand what I'm
doing though so will wait for you to really fix thing...

**Comment by gpspake** [comment]
Nice. Since Grunterie is a Foundation project, and you don't need to worry about losing changes from updating the theme, this is a case where you can make most of your changes from the parent theme. I still use a child theme for templates but I make all of my style edits in the main foundation project.

I've tried to reorganize everything to make this easy. You're already using Grunt so you shouldn't have much trouble...

One of the nice things about working with Foundation is you can edit most of the framework settings through the [settings.scss](https://github.com/gpspake/grunterie/blob/master/scss/_settings.scss) file. Just find what you want to change and uncomment the variable and add your preferred value. I actually prefer to leave the default value commented out and add the new one below it so I can always see what the default one was. I completely updated this file with this last Grunterie update to match the latest version of Foundation. It never gets overwritten, so it has to be updated manually.

All of the additional styles are broken down in to [partials](https://github.com/gpspake/grunterie/tree/master/scss/partials). Any styles should go in these files. I've included a [blank custom scss file](https://github.com/gpspake/grunterie/blob/master/scss/partials/_grunterie-custom.scss) for any new styles but you can edit the other partials too. You can also add new partials, in fact it's recommended, just keep the same naming convention as the others and make sure to import them in the [main stylesheet](https://github.com/gpspake/grunterie/blob/master/scss/style.scss) 

As far as the child theme goes, I think I'm going to simplify it so that there's just a single blank style.scss that get's loaded after the parent theme stylesheet. I almost want to discourage editing styles too much in the child theme since it's so much easier to edit foundation through the settings file and partials. 

Hope this helps. I should be updating it soon.

**Comment by titaniumbones** [comment]
Huh, cool.  So are you saying changes to  `scss/_settings.scss` in the parent theme are not tracked by git -- and ditto for `scss/partials/_custom.scss`? If true that's great.  Can I ask, how do you do that?  I don't see anything in .gitignore that would stop updates to either file...  Thanks again! -m

**Comment by gpspake** [comment]
Those are under revision control and if you're using git for your project I would definitely track those changes. I probably could have been more clear there. I just meant that those files won't be over written if you update your Foundation project... so if a new Foundation comes out, like 5.6, and changes are made to settings.scss, you just have to compare them to move any changes over. bower update will still pull in your dependencies but that settings.scss file is all yours so it won't change.

If I were to make changes to this repo and you pulled them down then, yes, that file would be updated but once you've gotten Grunterie set up, you shouldn't have to do that. If a new version of Foundation comes out, you can just update Foundation; you don't have to depend on Grunterie being up to date. All I really do when I update it is update the dependencies and run npm update and bower update. This one was just an exception because I wanted to re-organize some stuff.

So basically to update grunterie when a new version of Foundation comes out...
- create a new foundation project for reference (foundation new project --libsass)
- update dependencies in bower.json and package.json (Look at the one's in the new foundation project to see what they are)
- npm update
- bower update
- compare grunterie settings.scss with new foundation project settings.scss and move any new stuff over to the grunterie settings.scss

That's about it. Bower does most of the work. The only thing I really have to do manually is compare the settings.scss file.

**Comment by titaniumbones** [comment]
ah that's great, thank you.  cool.
